CHANGELOG.md

```diff
@@ -5,12 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+<!-- ## Unreleased -->
+<!-- Add new, unreleased changes here. -->
+
+## [3.0.0-pre.10] - 2018-02-01
 * [BREAKING] The `UrlResolver#relative()` method now returns a
```

Comment:
In the past we haven't considered TypeScript type changes breaking, provided that runtime behavior is unchanged.

Reply:
Okay.  Updating.